lib/src/core/utils/injections.dart

```diff
@@ -3,7 +3,7 @@ import 'package:flutter_clean_architecture/src/core/network/dio/dio_operations.d
 import 'package:flutter_clean_architecture/src/features/home/articles_injections.dart';
 import 'package:flutter_clean_architecture/src/shared/app_injections.dart';
 import 'package:get_it/get_it.dart';
-import 'package:secure_shared_preferences/secure_shared_preferences.dart';
+import 'package:secure_shared_preferences/secure_shared_pref.dart';
```

Comment:
why this changed bro? any issue

Reply:
getting error when latesh pull is taken, i will revert it